packages/browser/package.json

```diff
@@ -87,6 +88,7 @@
     "license-checker": "^25.0.1",
     "lodash.clonedeep": "^4.5.0",
     "node-jose": "^1.1.3",
+    "oidc-client": "^1.10.1",
```

Comment:
Instead of depending on `oidc-client`, this package should depend on `oidc-client-dpop-browser` (which exports `oidc-client` classes and functions)

Reply:
Sure absolutely, but let's do that on the next PR though (as it's not really directly relevant to this ticket).